### ISSUE TITLE
Exception when keyword is used in clojure.test/testing

### ISSUE THREAD

**Issue by cursive-ide** [cursive-ide/cursive#2857]
Using a test like:
```clojure
(deftest keyword-testing-context
  (testing :context
    (is (= 2 3))))
```
will result in an exception like:
```
java.lang.ClassCastException: class clojure.lang.Keyword cannot be cast to class java.lang.String (clojure.lang.Keyword is in unnamed module of loader com.intellij.ide.plugins.cl.PluginClassLoader @3c308f47; java.lang.String is in module java.base of loader 'bootstrap')
 at cursive.documentation$escape_html.invokeStatic (documentation.clj:73)
    cursive.documentation$escape_html.invoke (documentation.clj:73)
    cursive.testing.actions$contexts_list_string$fn__10610.invoke (actions.clj:353)
    clojure.core$map$fn__5935.invoke (core.clj:2772)
    clojure.lang.LazySeq.sval (LazySeq.java:42)
    clojure.lang.LazySeq.seq (LazySeq.java:51)
    clojure.lang.Cons.next (Cons.java:39)
    clojure.lang.RT.next (RT.java:771)
    clojure.core$next__5451.invokeStatic (core.clj:64)
    clojure.string$join.invokeStatic (string.clj:191)
    clojure.string$join.invoke (string.clj:180)
    cursive.testing.actions$contexts_list_string.invokeStatic (actions.clj:352)
    cursive.testing.actions$contexts_list_string.invoke (actions.clj:351)
```

**Comment by cursive-ide** [comment]
Fix released in 1.13.1-eap9 on 2023-11-27.

**Comment by cursive-ide** [comment]
Fix released in 1.13.1 on 2023-12-07.
